srcpkgs/mimalloc/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'mimalloc'
+pkgname=mimalloc
+version=1.7.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+short_desc="Compact general purpose allocator with excellent performance"
```

Comment:
Honestly, the libraries name suggested its `short_desc` should be: `Microsoft's malloc`.